coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -4607,6 +4607,10 @@ def _scatter(context, inputs, mode, name):
     axis = inputs[1].val
     indices = inputs[2]
     updates = inputs[3]
+    if not types.is_tensor(updates.sym_type):
```

Comment:
Rather than `not types.is_tensor(updates.sym_type)`, I think `types.is_scalar(updates.sym_type)` would be better.

Reply:
done